extensions/import/src/common/apiWrapper.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+/**
+ * Wrapper class to act as a facade over VSCode and Data APIs and allow us to test/mock callbacks into
+ * this API from our code
+ */
+
+export class ApiWrapper {
+
+	public createOutputChannel(name: string): vscode.OutputChannel {
```

Comment:
Whats the goal in introducing this? From what I can tell you are just adding integration tests, which makes this api wrapper unnecessary.

Reply:
The naming is kind of weird in the [test script](https://github.com/microsoft/azuredatastudio/pull/10529/files#diff-c309739b46f518698a67c1ad81d614cc)  but those are unit tests and not integration tests. I've created the wrapper just to mock those apis. Correct me if I'm wrong.

Reply:
What i'm saying is youy don't need to mock them. they aren't true unit tests in that they run in their own context. Those tests get run in a ADS environment, vscode, ads, all that will exist.

Reply:
@anthonydresser I think the plan is to test this feature without an active SQL connection, so I think some of the API calls will be mocked.

Reply:
It's a consistency thing too - just keeping all vscode/azdata calls through the api wrapper so that if we do care about mocking them out if we can. And adding them here doesn't really have any noticeable impact.